modules/apps/29-fee/keeper/escrow.go

```diff
@@ -26,18 +28,31 @@ func (k Keeper) EscrowPacketFee(ctx sdk.Context, identifiedFee types.IdentifiedP
 		return sdkerrors.Wrapf(types.ErrRefundAccNotFound, "account with address: %s not found", refundAcc)
 	}
 
-	coins := identifiedFee.Fee.RecvFee
-	coins = coins.Add(identifiedFee.Fee.AckFee...)
-	coins = coins.Add(identifiedFee.Fee.TimeoutFee...)
+	if k.channelKeeper.HasPacketAcknowledgement(ctx, identifiedFee.PacketId.PortId, identifiedFee.PacketId.ChannelId, identifiedFee.PacketId.Sequence) {
+		return sdkerrors.Wrapf(channeltypes.ErrAcknowledgementExists, "failed to escrow fee for packet sequence %d", identifiedFee.PacketId.Sequence)
+	}
 
-	if err := k.bankKeeper.SendCoinsFromAccountToModule(
-		ctx, refundAcc, types.ModuleName, coins,
-	); err != nil {
+	coins := identifiedFee.Fee.EscrowTotal()
```

Comment:
I've added a helper method `EscrowTotal()` to `Fee` to return the total escrowable amount as it was useful for keeping tests clean

Reply:
maybe it should just be `Total()`?